rules/web/web_cve_2022_27925_exploit.yml

```diff
@@ -0,0 +1,39 @@
+title: Zimbra Collaboration Suite Email Server Unauthenticated RCE
+id: dd218fb6-4d02-42dc-85f0-a0a376072efd
+description: Detects an attempt to leverage the vulnerable servlet "mboximport" for an unauthenticated remote command injection
+references:
+  - https://www.volexity.com/blog/2022/08/10/mass-exploitation-of-unauthenticated-zimbra-rce-cve-2022-27925/
+  - https://www.yang99.top/index.php/archives/82/
+  - https://github.com/vnhacker1337/CVE-2022-27925-PoC
+status: experimental
+Author: '@gott_cyber'
+date: 2022/08/17
+tags:
+  - attack.initial_access
+  - attack.t1190
+  - cve.2022.27925
+logsource:
+    category: webserver
+detection:
+    selection_servlet:
+        cs-method: 'POST'
+        c-uri|contains: '/service/extension/backup/mboximport?'
+    selection_uri:
+        c-uri|contains|all:
+            - 'account-name'
+            - 'ow'
+            - 'no-switch'
+            - 'append'
+        cs-cookie: 'ZM_AUTH_TOKEN'
```

Comment:
`cs-cookie` not sure we have this in the log.
If the ref you get  a "normal" web  log: 
[attacker_ip] - - [30/Jun/2022:05:33:18 +0000] "POST [snipped]/service/extension/backup/mboximport?account-name=admin&ow=1&no-switch=true HTTP/1.0" 401 299 "[snipped]" "Mozilla/5.0 (X11; Linux x86_64; rv:101.0) Gecko/20100101 Firefox/101.0"

Reply:
This is a good point. I ended up stripping the field.